projects/observability/src/shared/components/bar-gauge/bar-gauge.component.ts

```diff
@@ -48,9 +49,10 @@ import {
               [ngClass]="{ 'hide-last-divider': this.nearMaxValue }"
               [style.background]="segment.color"
               [style.width.%]="segment.percentage"
-              htTooltip="{{ segment.label }} : {{ segment.value | htDisplayNumber }}"
+              [htTooltip]="this.segmentsTooltip ? this.segmentsTooltip : plainTooltip"
```

Comment:
The context is needed otherwise you would only have a static template - same for every segment. If you want to define one per segment type, that would work without context.  Either way, should take the template as part of the segment to allow it to work either statically or dynamically

Reply:
> Depends on adding `[htTooltipContext]` as supported in #1464 CC @adisreyaj

Will be merged as soon as I add/update tests.